hi-ml/src/health_ml/utils/regression_test_utils.py

```diff
@@ -69,6 +66,174 @@ def _check_values_match(expected_v: Any, actual_v: Any, tolerance: float = 1e-5)
                     _check_values_match(expected_value, actual_value, tolerance=tolerance)
 
 
+def _compare_metric_values(expected: Any, actual: Any, tolerance: float = 1e-5) -> str:
+    """Compares an expected and an actual value coming from a metrics dictionary. If the values are numeric
+    and are close enough (within the given tolerance), an empty string is returned. String values
+    are always considered to match, because they usually contain AzureML resource ID. If the actual
```

Comment:
Why is the Azure ML resource ID returned in the metrics? Should we remove this in the function to get metrics?

Reply:
I've removed strings for now, regression tests only allow numeric values, which I think should be good enough